packaging/elasticsearch_plugin.py

```diff
@@ -111,30 +115,57 @@ def parse_plugin_repo(string):
 
     return repo
 
-
 def is_plugin_present(plugin_dir, working_dir):
     return os.path.isdir(os.path.join(working_dir, plugin_dir))
 
-
 def parse_error(string):
     reason = "reason: "
     try:
         return string[string.index(reason) + len(reason):].strip()
     except ValueError:
         return string
 
+def install_plugin(module, plugin_bin, plugin_name, version, url, proxy_host, proxy_port, timeout):
+    cmd_args = [plugin_bin, PACKAGE_STATE_MAP["present"], plugin_name]
 
-def main():
+    if version:
+        name = name + '/' + version
```

Comment:
@ThePixelDeveloper I now add version only during plugin installation because it's needed for plugin removal.